pyvista/plotting/plotting.py

```diff
@@ -272,7 +272,7 @@ def store_mouse_position(self, *args):
         self.mouse_position = self.iren.GetEventPosition()
 
     def store_click_position(self, *args):
```

Comment:
@banesullivan it would seem this function is also called in `picking.py` on line 348 in `pick_click_position` however I cannot see a definition of the function in `PickingHelper` if it is an abstract method perhaps raise a not implmented error

Reply:
So this gets into a rather nasty design choice I made a while back when implementing the `PickingHelper` class. This class is separated out mostly just to make managing the code easier because the `plotting.py` module has over 4000 lines of code at this point.

I should add abstract methods to the `PickingHelper` to make this implementation cleaner.

This will come in a separate PR and I'll try to do this with similar helper classes